SLHCUpgradeSimulations/Configuration/python/combinedCustoms.py

```diff
@@ -333,6 +338,98 @@ def cust_2023HGCalV6Muon(process):
     """
     process = cust_2023HGCal_common(process)
     process = customise_me0(process)
+    process=customisePostLS1(process)
```

Comment:
@lgray the vast majority of this section of additions is a duplicate of what's done by the `cust_2023HGCal_common(process)` call.  Is this intentional?  This could be the result of a git cockup doing the merge.
If it is intentional I would suggest allowing `cust_2023HGCal_common(process)` to do its thing and then just modify the one or two lines that are different.  As it currently stands I think it's confusing which section of HGCal customisation code does what.